app/src/main/java/net/gaast/giggity/Schedule.java

```diff
@@ -421,6 +424,155 @@ private void loadIcal(BufferedReader in) {
 		}
 	}
 
+	/*Reading the JSON open event format data fetched from the API end point*/
+	private void loadJson(BufferedReader in) {
+
+		StringBuffer buffer = new StringBuffer();
+		SimpleDateFormat df = new SimpleDateFormat();
+		HashMap<String, Schedule.Line> tentMap = new HashMap<String, Schedule.Line>();
+		Scanner s = new Scanner(in);
+
+		while (s.hasNext()) {
+			buffer.append(s.nextLine());
+
+		}
+
+		String output = buffer.toString();
+		Schedule.Line line = null;
+
+		try {
+
+			JSONObject conference = new JSONObject(output);
```

Comment:
Yes @Wilm0r . It uses library named "simple json parser". I try to avoid using external library if things are happening to keep up with the app's size. But it's okay if you feel like it. I will help in making changes with another library once this gets merged.